### PR TITLE
Create Rule11-16.md

### PR DESCRIPTION
**Pull request by KarenWGard**

### CODE REVIEW

File: docs/section11/Rule11-16.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  - otherwise, continue to rule logic: `
+  - `CONTINUE TO RULE LOGIC`
+    ## Rule Logic:
+    - create variable heater_fuel_type: `heater_fuel_type = swh_eq.heater_fuel_type`
+    **Rule Assertion:**  
+    - CASE 1: if the fuel source is NATURAL_GAS, then PASS: `if heater_fuel_type == "NATURAL_GAS": PASS`
+    - Case 2: Else if the fuel source is propane: `elif: heater_fuel_type == "PROPANE": UNDETERMINED and raise_message "BASELINE SERVICE WATER HEATING EQUIPMENT SOURCE IS MODELED AS PROPANE. VERIFY IF NATURAL GAS IS NOT AVAILABLE FOR THE PROPOSED BUILDING SITE AS DETERMINED BY THE RATING AUTHORITY."`
+    - Case 3: All others fail: `else: FAIL`
+
+     **Rule Assertion - Zone:**
```
Comment (claperle) [inline docs/section11/Rule11-16.md:29]: Not following why there ate two rule assertion sections. I think when we did this check for HVAC we also checked the proposed design model to see if there is natural gas. If there is then we would not expect propane in the baseline.

File: docs/section11/Rule11-16.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# ServiceWaterHeating - Rule 11-16
+
+**Schema Version:** 0.0.36
```
Comment (JacksonJ-KC) [inline docs/section11/Rule11-16.md:3]: most recent version is 0.0.37

File: docs/section11/Rule11-16.md (hunk 1 above)
Comment (JacksonJ-KC) [reply docs/section11/Rule11-16.md:29]: Seems like this `Rule Assertion - Zone` section should be removed.

File: docs/section11/Rule11-16.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+**Evaluation Context:** Each SWH Equipment  
+**Data Lookup:**   
+**Function Call:**
+
+
+**Applicability Checks:**
+- look at each SHW Equipment: `for swh_eq in B_RMD.service_water_heating_equipment:`
+  - if the fuel source is electricity, the rule is not applicable: `if shw_eq.heater_fuel_type == "ELECTRICITY": NOT_APPLICABLE`
+  - otherwise, continue to rule logic: `
+  - `CONTINUE TO RULE LOGIC`
```
Comment (JacksonJ-KC) [inline docs/section11/Rule11-16.md:21]: remove newline to bring this up with its plain-english partner

File: docs/section11/Rule11-16.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    ## Rule Logic:
+    - create variable heater_fuel_type: `heater_fuel_type = swh_eq.heater_fuel_type`
+    **Rule Assertion:**  
+    - CASE 1: if the fuel source is NATURAL_GAS, then PASS: `if heater_fuel_type == "NATURAL_GAS": PASS`
+    - Case 2: Else if the fuel source is propane: `elif: heater_fuel_type == "PROPANE": UNDETERMINED and raise_message "BASELINE SERVICE WATER HEATING EQUIPMENT SOURCE IS MODELED AS PROPANE. VERIFY IF NATURAL GAS IS NOT AVAILABLE FOR THE PROPOSED BUILDING SITE AS DETERMINED BY THE RATING AUTHORITY."`
```
Comment (JacksonJ-KC) [inline docs/section11/Rule11-16.md:26]: Per Christina's comment below, we can further evaluate a subset of these UNDETERMINED cases - those where the proposed fuel type is not "PROPANE" can be FAIL outcomes

Comment (JacksonJ-KC) [inline docs/section11/Rule11-16.md:26]: Suggest changing the message to confirm that both the baseline and proposed where modeled using propane.

File: docs/section11/Rule11-16.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+     **Rule Assertion - Zone:**
+    - Case1: rule_status is PASS: `if rule_status == "PASS": PASS`.
+    - Case2: rule_status is UNDETERMINED, UNDETERMINED & return rule_note: `elsif rule_status == "UNDETERMINED": UNDETERMINED; rule_note`
+    - Case3: rule_status is FAIL, FAIL & return rule_note: `elsif rule_status == "FAIL": FAIL; rule_note`
+
+
+**Notes:**
+
+1. original language specified gas storage water heaters only.  The rule applies only to gas storage water heaters because "storage" type water heaters are the only valid baseline fossil-fuel water heaters.  However, the heater type is checked in another rule.  Can we just check fuel-type here?
```
Comment (JacksonJ-KC) [inline docs/section11/Rule11-16.md:37]: Makes sense to me to evaluate for all non-electric water heaters regardless of whether it is a storage type

File: docs/section11/Rule11-16.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+**Applicability Checks:**
+- look at each SHW Equipment: `for swh_eq in B_RMD.service_water_heating_equipment:`
+  - if the fuel source is electricity, the rule is not applicable: `if shw_eq.heater_fuel_type == "ELECTRICITY": NOT_APPLICABLE`
+  - otherwise, continue to rule logic: `CONTINUE TO RULE LOGIC`
+  
+    ## Rule Logic:
+    - create variable heater_fuel_type: `heater_fuel_type = swh_eq.heater_fuel_type`
+    - create a list that will be used to store the fuel types used in the proposed model swh equipment: `proposed_swh_eq_fuel_types = []`
+    - look at each swh equipment in the proposed model and collect a list of fuel types: `for swh_eq_p in P_RMD.service_water_heating_equipment: if !(swh_eq_p.fuel_type.in? proposed_swh_eq_fuel_types): proposed_swh_eq_fuel_types.append(swh_eq_p.fuel_type)`
```
Comment (claperle) [inline docs/section11/Rule11-16.md:25]: I recommend that the RDS look at all equipment. If gas is used for space heating equipment or miscellaneous equipment then that would tell us it is available on site.

Comment (claperle) [reply docs/section11/Rule11-16.md:25]: Also, looks like some Ruby syntax was used.

File: docs/section11/Rule11-16.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+    **Rule Assertion:**  
+    - CASE 1: if the fuel source is NATURAL_GAS, then PASS: `if heater_fuel_type == "NATURAL_GAS": PASS`
+    - Case 2: Else if the fuel source is propane and the proposed fuel source is propane: `elif: heater_fuel_type == "PROPANE" && len(proposed_swh_eq_fuel_types) == 1 && proposed_swh_eq_fuel_types[0] == "PROPANE": UNDETERMINED and raise_message "Both baseline and proposed service water heating equipment is modeled as propane.  Verify that natural gas is not available for the proposed building site as determined by the rating authority."`
```
Comment (claperle) [inline docs/section11/Rule11-16.md:29]: Electricity could also be a proposed fuel type, not following why LEN would have to be 1

File: docs/section11/Rule11-16.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+    - Case 3: Else if the fuel source is propane and any of the proposed fuel sources is NATURAL_GAS, then FAIL: `if fuel_source == "PROPANE" && "NATURAL_GAS".in? proposed_swh_eq_fuel_types: FAIL and raise_message "The fuel source for the baseline is propane, however the fuel source for the proposed is Natural Gas.  When natural gas is available on-site, natural gas is the required fuel source for the baseline model."'
```
Comment (claperle) [inline docs/section11/Rule11-16.md:30]: Ruby syntax

File: docs/section11/Rule11-16.md [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+    - Case 4: Else if the fuel source is propane, then UNDETERMINED: `if heater_fuel_type == "PROPANE": UNDETERMINED`
```
Comment (claperle) [inline docs/section11/Rule11-16.md:31]: Include message to explain what the UNDETERMINED outcome means "The fuel source for the baseline is propane.  When natural gas is available on-site, natural gas is the required fuel source for the baseline model. Verify that natural gas is not availabile on-site."

File: docs/section11/Rule11-16.md (hunk 7 above)
Comment (claperle) [reply docs/section11/Rule11-16.md:29]: Oh okay, sounds good.

Comment (KarenWGard) [reply docs/section11/Rule11-16.md]: updated to have just one result for propane fuel source

**Review by claperle — APPROVED** [review]

File: docs/section11/Rule11-16.md [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+
+
+**Applicability Checks:**
+- look at each SHW Equipment: `for swh_eq in B_RMD.service_water_heating_equipment:`
+  - if the fuel source is electricity, the rule is not applicable: `if shw_eq.heater_fuel_type == "ELECTRICITY": NOT_APPLICABLE`
+  - otherwise, continue to rule logic: `CONTINUE TO RULE LOGIC`
+  
+    ## Rule Logic:
+    - create variable heater_fuel_type: `heater_fuel_type = swh_eq.heater_fuel_type`
+    - use the function get_fuels_modled_in_RMR to get a list of the fuels used in the proposed model: `proposed_fuels = get_fuels_modeled_in_RMR(P_RMD)`
```
Comment (JacksonJ-KC) [inline docs/section11/Rule11-16.md:25]: ~~RMR~~ RMD

File: docs/section11/Rule11-16.md [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+
+    **Rule Assertion:**  
+    - CASE 1: if the fuel source is NATURAL_GAS, then PASS: `if heater_fuel_type == "NATURAL_GAS": PASS`
+    - Case 2: Else if the fuel source is propane and any of the proposed fuel sources is NATURAL_GAS, then FAIL: `if fuel_source == "PROPANE" && "NATURAL_GAS".in proposed_fuels: FAIL and raise_message "The fuel source for the baseline is propane, however the fuel source for the proposed is Natural Gas.  When natural gas is available on-site, natural gas is the required fuel source for the baseline model."'
```
Comment (JacksonJ-KC) [inline docs/section11/Rule11-16.md:29]: I think missing back tick ` to close the line is causing format issue

**Review by JacksonJ-KC — APPROVED** [review]

**Pull request merged**